code/modules/jobs/job_types/agent.dm

```diff
@@ -82,19 +82,19 @@
 		facility_full_percentage = 100 * (SSabnormality_queue.spawned_abnos / SSabnormality_queue.rooms_start)
 	// how full the facility is, from 0 abnormalities out of 24 cells being 0% and 24/24 cells being 100%
 	switch(facility_full_percentage)
-		if(15 to 29) // Shouldn't be anything more than TETHs (4 Abnormalities)
+		if(15 to 29.99) // Shouldn't be anything more than TETHs (4 Abnormalities)
```

Comment:
You can do 15 to 29 here, and then 29 to 44 on the next one.
Sadly I can't approve this as it doesn't follow conventions </3

Reply:
Oh, it does? neat! Fixed the code to what you said.